### PR TITLE
Uiux/suspensions wording

### PR DESCRIPTION
**Pull request by yachtcaptain23**
Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave-intl/publishers/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Tagged reviewers.
- [ ] Integrated piwik/matomo (for code that adds new buttons).
- [ ] Addressed or ignored all brakeman warnings

Test Plan:

Reviewer Checklist:

Tests
- [ ] Adequate test coverage exists to prevent regressions

Security:
- [ ] No raw SQL -- Always prefer ActiveRecord query helpers ([more info on StackOverflow](https://stackoverflow.com/questions/41410752/rails-5-sql-injection#41452695))
- [ ] XSS is mitigated -- Avoid `html_safe` and `raw`; escape untrusted content from users and 3rd party APIs ([Rails XSS guide](https://brakemanpro.com/2017/09/08/cross-site-scripting-in-rails) and [OWASP XSS Prevention Cheat Sheet](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet))

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -456,8 +456,13 @@ en:
       download: Download
       statement_file_name: BravePaymentsStatement
     suspended_error:
-      heading_html: Due to suspicious activity,<br/> your account is suspended.
-      body: Please contact support+publishers@basicattentiontoken.org if you believe that activities on your account are not aberrant or irregular.
+      heading_html: Your Account Is Under Review
+      body: >
+        We try to keep the BAT User Growth Pool (UGP) fair for all parties involved via publishers.basicattentiontoken.org. However, the UGP is constantly a target for a lot of unnatural gain and activities, and we, unfortunate
```
Comment (corymcdonald) [inline config/locales/en.yml]: Did this sentence get cut off?

File: config/locales/en.yml [hunk 2]
```diff
@@ -456,8 +456,13 @@ en:
...
+        <br/><br/>
+        We know that this is inconvenient and we're sorry about that. But we'd rather play it safe than sorry when it comes to protecting your account and Brave Payments.
+        <br/><br/>
+        If you think you know what might be going on, the fastest way to clear this up is to get in touch with us at support+publishers@basicattentiontoken.org where a real actual human will read any reply to your message — normally in 48 hours.
```
Comment (corymcdonald) [inline config/locales/en.yml]: Think we should adjust this to be "normally within 48 hours"?

**Comment by corymcdonald** [comment]
Should we make the `support+publishers@basicattentiontoken.com` a `mailto:` link?

**Comment by yachtcaptain23** [comment]
screen shot 2018-11-27 at 11 30 24 am

**Comment by yachtcaptain23** [comment]
screen shot 2018-11-27 at 11 37 26 am

**Pull request merged**